### PR TITLE
updated conda installation command

### PR DESCRIPTION
**Pull request by Agent007**
Hi @rbharath , I've updated the README to reflect the conda installation as you requested.

### CODE REVIEW

**Comment by rbharath** [comment]
@Agent007, thanks for your first DeepChem PR!

Could you restore the deleted lines and add the Omnia install command in a new section?

**Comment by Agent007** [comment]
@rbharath Done. Hope it looks good this time.

File: README.md [hunk 1]
```diff
@@ -63,6 +63,11 @@ Check [this link](https://conda.io/docs/using/envs.html) for more information ab
 the benefits and usage of conda environments. **Warning**: Segmentation faults can [still happen](https://github.com/deepchem/deepchem/pull/379#issuecomment-277013514)
 via this installation procedure.
 
+### Easy Install via Conda
+```bash
+conda install -c omni deepchem
```
Comment (rbharath) [inline README.md]: s/omni/omnia/

**Comment by rbharath** [comment]
Almost there! Just a typo to fix this time.

**Comment by Agent007** [comment]
@rbharath Fixed!

**Comment by rbharath** [comment]
Great, thanks! Will just wait for Travis to finish running.

**Comment by rbharath** [comment]
LGTM!

**Pull request merged**